modules/setting/setting.go

```diff
@@ -161,6 +161,7 @@ var (
 		PullRequestQueueLength int
 		PreferredLicenses      []string
 		DisableHTTPGit         bool
+		ForceCloneSSHURL       bool
```

Comment:
This property name is misleading too because it could also be the forced clone ssh url (developers who are not involved in this issue may have problems with this). I would use `UseSSHURIScheme`.